### ISSUE TITLE
Class constructors cannot be invoked without new

### ISSUE THREAD

**Issue by robin-barnes-triptease** [vdemedes/mongorito#62]
On the latest version, when doing this:

````javascript
import { Model } from 'mongorito';

class Bar extends Model {

}

new Bar();
````

I get this:

````
/Users/robinbarnes/projects/config-service/node_modules/mongorito/lib/mongorito.js:140
  constructor (attrs, options) {
              ^

TypeError: Class constructors cannot be invoked without 'new'
````

**Comment by vdemedes** [comment]
Are you using master branch or installing from npm?

**Comment by robin-barnes-triptease** [comment]
When using npm:     "mongorito": "^1.3.4",

It works if using node I do:

````javascript
'use strict';

var Model = require('mongorito').Model;

class Bar extends Model {

}

new Bar();
````

It's with babel-node it doesn't work.

**Comment by vdemedes** [comment]
Fix is pushed to npm, available as `mongorito@1.4.0`.

**Comment by robin-barnes-triptease** [comment]
Thanks!
